github/src/main/kotlin/gropius/sync/github/Incoming.kt

```diff
@@ -238,56 +230,33 @@ class Incoming(
         issueGrabber.iterate {
             issueModified(imsProjectConfig, it)
         }
-        for (issue in issueInfoRepository.findByUrlAndDirtyIsTrue(imsProjectConfig.url).toList()) {
-            val timelineGrabber = TimelineGrabber(
-                issueInfoRepository, mongoOperations, issue.githubId, apolloClient, imsProjectConfig
-            )
-            timelineGrabber.requestNewNodes()
+        for (issueInfo in issueInfoRepository.findByUrlAndDirtyIsTrue(imsProjectConfig.url).toList()) {
```

Comment:
this function is way too long

Reply:
66490fcc1a92626cad40681eb0bcd95e2484c9a4